index.html

```diff
@@ -46,102 +46,17 @@ <h2>LET'S CONNECT</h2>
         <div class="popup"></div>
       </div>
 
-      <div class="grid_desktop grid_card">
-        <div class="img-container">
-          <img src="./Asset/arteto.svg" alt="" />
-        </div>
-        <div class="project-info">
-          <h2>Tonic</h2>
-          <ul class="dcan">
-            <li class="dcanopy">CANOPY</li>
-            <li class="dbackend">Back End Dev</li>
-            <li class="dtwenty">2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="dlang">
-            <li class="dhtml">HTML</li>
-            <li class="dcss">CSS</li>
-            <li class="djs">JavaScript</li>
-          </ul>
-          <button class="dseepro" id="popD">See project</button>
-        </div>
+      <div class="grid_card">
+        <img class="one" src="./Asset/arteto.svg" alt="" />
```

Comment:
- [ ] when opening the mobile menu the text on the background is on top of the the menu please make sure it is behind the menu and that it is blurred similar to the template:
your project:
![afbeelding](https://user-images.githubusercontent.com/39240820/232522044-7476dd62-3c7b-4686-a6b8-b0b776f99fff.png)
Figma template:
![afbeelding](https://user-images.githubusercontent.com/39240820/232522156-f96819dd-f0a1-4811-a3e2-8e13d65cbd7a.png)